frontend/test/__support__/e2e/commands/api/question.js

```diff
@@ -1,51 +1,92 @@
+Cypress.Commands.add("createQuestion", (questionDetails, customOptions) => {
```

Comment:
The biggest question for me at this stage is whether to use named parameters here or not.
If we decide to do so, any previous reference to the `cy.createQuestion` and `cy.createNativeQuestion` need to be updated. And we've been using these custom commands a looot.